admin/static/js/metrics/metrics.js

```diff
@@ -1,187 +1,499 @@
-"use_strict";
+'use strict';
 
+require('c3/c3.css');
+require('keen-dataviz/dist/keen-dataviz.min.css');
 
+
+var c3 = require('c3/c3.js');
 var keen = require('keen-js');
+var keenDataviz = require('keen-dataviz');
+var keenAnalysis = require('keen-analysis');
 
 
-var client = new Keen({
+var client = new keenAnalysis({
     projectId: keenProjectId,
-    readKey : keenReadKey
+    readKey: keenReadKey
 });
 
+var publicClient = new keenAnalysis({
+    projectId: keenPublicProjectId,
+    readKey: keenPublicReadKey
+});
 
-Keen.ready(function () {
+var defaultHeight = 200;
+var bigMetricHeight = 350;
+var institutionTableHeight = "auto";
 
-    // Active user count!
-    var active_user_count = new Keen.Query("sum", {
-        eventCollection: "user_summary",
-        targetProperty: "status.active",
-        timeframe: "previous_1_days",
-        timezone: "UTC"
+var defaultColor = '#00BBDE';
```

Comment:
Any way these can be pulled from CSS? If not, not a big deal, but it might also be better to make these configurable. You know, just in case we change our color scheme or something on the admin app.